kernel/kernel-api/src/main/java/io/delta/kernel/internal/replay/ConflictChecker.java

```diff
@@ -226,6 +236,10 @@ public CloseableIterable<Row> getUpdatedDataActions() {
     public List<DomainMetadata> getUpdatedDomainMetadatas() {
       return updatedDomainMetadatas;
```

Comment:
Please update this method to fetch from CRC (if available) in your DM loading from CRC PR.

Reply:
Actually this is the domainMetadataToCommit for losing txn, so should not come from CRC. I will rename these confusing vars and method in the PR